test/integration/helpers/screenshot.js

```diff
@@ -0,0 +1,11 @@
+'use strict';
+
+var fs = require('fs');
+
+exports.take = function(savePath) {
+  return browser.takeScreenshot().then(function(png) {
+    var stream = fs.createWriteStream(savePath || '/tmp/origin-web-console-e2e-screenshot.png');
```

Comment:
we should avoid taking screenshots during tests except when failures occur, can you add a comment in here that says as much.  Screenshots are taken automatically today when protractor sees a failure so you need a good reason to explicitly take another screenshot.  Obviously also fine to use this for debugging purposes while writing tests as long as the calls are taken out before its merged